src/js/common/schemaform/review/ObjectField.jsx

```diff
@@ -102,13 +102,14 @@ class ObjectField extends React.Component {
 
     const renderedProperties = this.orderAndFilterProperties(properties)
       .map((objectFields, index) => {
-        const firstField = objectFields[0];
         // show all the fields only if the first one is truthy, since more than one field
         // means this is an expanding group
-        if (objectFields.length > 1 && !!formData[firstField]) {
+        const [first, ...rest] = objectFields;
```

Comment:
This logic copied from the normal ObjectField, which is what should have happened when I originally wrote it.